### PR TITLE
Rickie styleguide 3 16

### PR DESCRIPTION
**Pull request by runnabro**

### CODE REVIEW

File: src/html/logo.html [hunk 1]
```diff
@@ -2,38 +2,45 @@
...
 
     <figure class="figure figure-logo-lockup">
-      <img class="img" src="images/angular/runnable-logo-gray.svg" width="290px" alt"Lockup on Purple">
+      <img class="img" src="images/logo/runnable-logo-gray/runnable-logo-gray.svg" width="300px" alt"Lockup on White">
       <figcaption class="figcaption">Lockup on White</figcaption>
-      <a class="link" href="images/angular/runnable-icon-gray.svg" download>SVG</a>
+      <a class="btn btn-xs white" href="images/logo/runnable-logo/runnable-log.eps" download>Download EPS</a>
+      <a class="btn btn-xs white" href="images/logo/runnable-logo/runnable-logo.png" download>Download PNG</a>
+      <a class="btn btn-xs white" href="images/logo/runnable-logo/runnable-logo.svg" download>Download SVG</a>
+      <a class="btn btn-xs white" href="images/logo/runnable-logo/runnable-logo.zip" download>Download ZIP</a>
```
Comment (runnabro) [inline src/html/logo.html]: Let's remove the .zip

**Comment by rickiesherman** [comment]
@runnabro Going through this to look for errors

File: src/styles/scss/figures.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
   border-radius: 3px;
   margin: 6px 3px;
   padding: 15px;
-
+  vertical-align: top;
   .iconnables {
```
Comment (runnabro) [inline src/styles/scss/figures.scss]: Should have a line-break before new selectors

File: src/styles/scss/figures.scss [hunk 3]
```diff
@@ -68,7 +68,8 @@
...
 
 .figcaption {
-  color: $gray-light;
-  font-style: italic;
+  color: $gray-dark;
+  font-weight: bold;
   margin-top: 15px;
-}
\ No newline at end of file
+  margin-bottom: 10px;
```
Comment (runnabro) [inline src/styles/scss/figures.scss:75]: Alphabetical order

**Comment by runnabro** [comment]
There are still bear images that are not used here.

**Pull request merged**